src/api/bot.ts

```diff
@@ -71,6 +36,49 @@ router.post<{}>("/start", async (req, res, next) => {
       mentioned_handle: details?.metadata?.twitter_handle || "nerobossai",
       prompt: details.replies_prompt,
     });
+
+    if (details.nftCollection) {
+      nftcreationqueue.push({
+        mentioned_handle: details?.metadata?.twitter_handle || "nerobossai",
```

Comment:
let's create constant on top or create some default config for constants. eg: nerobossai here